README.rst

```diff
@@ -220,12 +220,12 @@ The image configuration options
 
 android-sparse
 **************
-Generate android sparse images. The are typically used by fastboot. Sparse
+Generate android sparse images. They are typically used by fastboot. Sparse
 images encode "don't care" areas and areas that are filled with a single
 32 bit value. As a result, they are often much smaller than raw disk
 images.
 Genimage assumes that all 'holes' in the input file are "don't care" areas.
-This is a reasonable assumption: Tools to generate filesystems typically
+This is a reasonable assumption: tools to generate filesystems typically
```

Comment:
I'm pretty sure nouns are capitalized after a colon.

Reply:
I do not have a strong position to that and will push a new revision where this one is rolled back.